hphp/runtime/ext/pdo_mysql/pdo_mysql.cpp

```diff
@@ -1281,7 +1285,11 @@ bool PDOMySqlStatement::getColumnMeta(int64_t colno, Array &ret) {
     ret.set(s_native_type, str);
   }
   ret.set(s_flags, flags);
+  ret.set(s_name, String(F->name, CopyString));
+  ret.set(s_org_name, String(F->org_name, CopyString));
   ret.set(s_table, String(F->table, CopyString));
+  ret.set(s_org_table, String(F->org_table, CopyString));
+  ret.set(s_length, F->length);
```

Comment:
`org_name`, `org_table` fields look like available since MySQL 4.0.3 merge. https://github.com/mysql/mysql-server/commit/7134ffec210edde21860a2b2c2654be481de49b4#diff-6391e1a12cc78a7dcd7a52a1fbb713fcR70

However not implemented in PHP yet. We can decision to implement what name used for this fields. There are some choices in my mind.

- Leave the property names like now.
- Change property names to common name like `original_name`, `original_table`
- Add prefix to property names like `hh_org_name`, `hh_org_table`
- Else?

But it should not matter because PHP documentation says that:
> This function is EXPERIMENTAL. The behaviour of this function, its name, and surrounding documentation may change without notice in a future release of PHP. This function should be used at your own risk.

http://php.net/manual/en/pdostatement.getcolumnmeta.php